magicblock-pubsub/src/pubsub_service.rs

```diff
@@ -89,6 +96,10 @@ impl PubsubService {
             |context: &RequestContext| Arc::new(Session::new(context.sender()));
 
         ServerBuilder::with_meta_extractor(self.io, extractor)
+            // NOTE: we just set the max number of allowed connections to a reasonably high value
+            // to satisfy most of the use cases, however this number cannot be arbitrarily large
+            // due to the preallocation involved, and a large value will trigger an OOM Kill
+            .max_connections(self.config.max_connections)
```

Comment:
Should we run a verification step over this config value to ensure that the max reasonable value isn't exceeded?

Reply:
No need, it will OOM anyway. Ideally we need custom pubsub implementation which doesn't rely on the weirdness of the third party lib